docs/getting-started-kubernetes.rst

```diff
@@ -1343,4 +1343,276 @@ Network Operator deployment with:
       volumes:
        - name: hugepage
          emptyDir:
-           medium: HugePages
\ No newline at end of file
+           medium: HugePages
+
+----------------------------------------------------
+Network Operator Deployment and OpenvSwitch offload
+----------------------------------------------------
+
+.. warning:: This feature is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+
+.. warning:: This mode of operation is not compatible with OFED container
+
+.. warning:: Tech Preview feature
+
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Network operator configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Deploy network-operator by Helm with sriov-network-operator and nv-ipam.
+
+
+``values.yaml``
+
+.. code-block:: yaml
+
+    sriovNetworkOperator:
+      enabled: true
+    deployCR: true
+    nvIpam:
+      deploy: true
+
+
+Enable manageSoftwareBridges featureGate for sriov-network-operator
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "manageSoftwareBridges": true  } } }' --type='merge'
+    
+Create IPPool object for nv-ipam
+
+.. code-block:: yaml
+
+    apiVersion: nv-ipam.nvidia.com/v1alpha1
+    kind: IPPool
+    metadata:
+      name: pool1
+      namespace: network-operator
+    spec:
+      subnet: 192.168.0.0/16
+      perNodeBlockSize: 100
+      gateway: 192.168.0.1
+      nodeSelector:
+        nodeSelectorTerms:
+        - matchExpressions:
+            - key: node-role.kubernetes.io/worker
+              operator: Exists
+
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Prerequisites for worker nodes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Supported operating systems:
+
+* Ubuntu 22.04
+
+OpenvSwitch from the NVIDIA DOCA package should be installed on each worker node.
```

Comment:
Instead of "NVIDIA DOCA package" let's use "NVIDIA DOCA for Host package with **doca-all** or **doca-networking** profile.

Reply:
done